os/net/ipv6/uipopt.h

```diff
@@ -547,6 +531,19 @@ void uip_log(char *msg);
  */
 #define UIP_TC_MAC_TRANSMISSION_COUNTER_MASK 0x3F
 
+#ifdef UIP_CONF_TAG_TC_WITH_VARIABLE_RETRANSMISSIONS
+#define UIP_TAG_TC_WITH_VARIABLE_RETRANSMISSIONS UIP_CONF_TAG_TC_WITH_VARIABLE_RETRANSMISSIONS
+#else
+#define UIP_TAG_TC_WITH_VARIABLE_RETRANSMISSIONS 1
```

Comment:
Maybe this should in fact be zero. As this feature will change traffic class as soon as we control the transmission count from uipbuf.

Reply:
I also vote for disabling by default, because this is anyway unused by default, is a bit experimental, and not really a standard feature